docs/pages/enroll-resources/application-access/cloud-apis/aws-console.mdx

```diff
@@ -780,6 +780,20 @@ instance and attach an IAM role to it.
 
 (!docs/pages/includes/aws-no-credential-provider.mdx service="Application"!)
 
+### Missing X-Forwarded-Host HTTP header
+
+You may enconter the error `proxied requests must include X-Forwarded-Host header`
+when making requests to your `tsh` local proxy. This is due to Telepor's
+expecting the HTTP requests to include the `X-Forwarded-Host` header, containing
```

Comment:
```suggestion
when making requests to your `tsh` local proxy. This is because Teleport
expects the HTTP requests to include the `X-Forwarded-Host` header, containing
```